addons/website_slides/controllers/main.py

```diff
@@ -466,7 +466,7 @@ def channel(self, channel, category=None, tag=None, page=1, slide_type=None, unc
         values['category_data'] = channel._get_categorized_slides(
             domain, order,
             force_void=not category,
-            limit=False if channel.channel_type != 'documentation' else self._slides_per_page if category else self._slides_per_category,
+            limit=False if channel.channel_type != 'documentation' or not values['channel'].slide_category_ids else self._slides_per_page if category else self._slides_per_category if not uncategorized else False,
```

Comment:
This has become impossible to read.
Can you make a variable with a clear if / elif statement so we can figure out the limit without having to read this 5 times while crying?